include/boost/json/string.hpp

```diff
@@ -2724,52 +2833,258 @@ class string
         InputIt first2,
         InputIt last2);
 
+    /** Replace a substring with a string.
+
+        Replaces `rcount` characters starting at index `pos` with those of
+        `[s, s + count2)`, where `rcount` is `std::min(count, size() - pos)`.
```

Comment:
oh... I see. It is a bit confusing. How about just "Replaces characters starting at index `pos`..." and then "The number of characters replaced will be equal to `std::min(count, size() - pos)` ?

Reply:
The reason I use that notation is to avoid repetition. I use `rcount` later in that javadoc to specify when exceptions are thrown.